app/css/ui-kit/colors.css

```diff
@@ -137,7 +137,7 @@
 
 .theme-light {
     --backgroundColorA: var(--c-FFFFFF);
-    --backgroundColorA-RGB: 255, 255, 255;
+    --backgroundColorA-RGB: 255 255 255;
```

Comment:
@dem4ron See https://tailwindcss.com/docs/customizing-colors#using-css-variables to understand these changes.